src/shogun/kernel/ShiftInvariantKernel.h

```diff
@@ -120,10 +122,14 @@ class ShiftInvariantKernel: public Kernel
 		return "ShiftInvariantKernel";
 	}
 
+	virtual SGMatrix<float64_t>
+	sample_spectral_density(int32_t dim_output, int32_t dim_input_space) const;
+
 protected:
 	/**
-	 * Computes distance between features a and b, where idx_{a,b} denote the indices
```

Comment:
just revert this (or don't commit it in the first place :))

Reply:
could you remove this change

Reply:
If you're talking of docs, I haven't changed anything. Can you try scrolling down?

Reply:
In general, try to avoid such whitespace changes in such PRs, they make reviewing more tedious. I can see that you didnt change anything. Just asking to not commit this no-op changes and send a whitespace cleanup PR later if you want